### PR TITLE
Mainly add support for vaillant B521 service command

### PR DESCRIPTION
**Pull request by tobylorenz**

### CODE REVIEW

**Comment by tobylorenz** [comment]
This change is regarding issue 1061 in the main repository: https://github.com/john30/ebusd/issues/1061.

File: ebusd-2.1.x/de/vaillant/e0.omu.csv [hunk 1]
```diff
@@ -1,85 +1,111 @@
 # type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
 #,OMU00,OMU_no_serial,OMU00 252,,,,,,,,,,
+# B509 service commands
```
Comment (john30) [inline ebusd-2.1.x/de/vaillant/e0.omu.csv:3]: english version is missing

**Review by john30 — CHANGES_REQUESTED** [review]

**Comment by john30** [comment]
reworked+merged in 679f14cf

**Comment by tobylorenz** [comment]
Yes, I'll monitor it.

**Pull request closed without merging**